os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -55,7 +55,7 @@
 #ifndef CONFIG_ENABLE_P2P
 #define CONFIG_ENABLE_P2P 0
 #endif
-#define SCAN_WITH_SSID 0
+#define SCAN_WITH_SSID 1

```

Comment:
How about adding `#define SCAN_WITH_SSID` to Kconfig to make user enable or disable this functionality via menuconfig without finding and modifying this file?
If then, you can change `#if SCAN_WITH_SSID` to `#ifdef SCAN_WITH_SSID` in line 800.